cmd/minishift/main.go

```diff
@@ -18,8 +18,10 @@ package main
 
 import (
 	"github.com/minishift/minishift/cmd/minishift/cmd"
+	"github.com/minishift/minishift/pkg/minikube/machine"
 )
 
 func main() {
+	machine.StartDriver()
```

Comment:
well, the review itself was messy as it contained incomplete merge actions. However, the changeset that remained was quite large. Just wonder how this made things work on macOS.